calico_containers/calicoctl.py

```diff
@@ -57,12 +57,12 @@
 from netaddr.core import AddrFormatError
 from prettytable import PrettyTable
 
-from calico_containers.adapter.datastore import (ETCD_AUTHORITY_ENV,
```

Comment:
LMK if this isn't true, I spent a bunch of time fiddling with python path issues in the build to get this working, so I'd like to know what went wrong if it did...